contrib/keyd/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "keyd"
+pkgver = "2.4.3"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_use_env = True
+hostmakedepends = ["gmake"]
+makedepends = ["linux-headers"]
+pkgdesc = "Key remapping daemon for linux"
+maintainer = "feurry <=@gmail.com>"
```

Comment:
are you `=@gmail.com`? (it has to be a valid email that isn't someone elses)

Reply:
thanks. fixed.